common/flavorization/00_CTP.txt

```diff
@@ -6868,6 +6868,1335 @@ duke_tribal_female_italian = {
+	tier = county
+	priority = 18
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+count_feudalclan_female_lombard = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 18
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+county_feudalclan_lombard = {
+	type = title
+	priority = 18
+	tier = county
+	cultures = { lombard }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+count_feudalclan_male_lombard_independent = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 19
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+count_feudalclan_female_lombard_independent = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = 19
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+county_feudalclan_lombard_independent = {
+	type = title
+	priority = 19
+	tier = county
+	cultures = { lombard }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+
+duke_feudalclan_male_lombard = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duke_feudalclan_female_lombard = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 28
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duchy_feudalclan_lombard = {
+	type = title
+	priority = 28
+	tier = duchy
+	cultures = { lombard }
+	governments = { feudal_government clan_government }
+	top_liege = no
+}
+duke_feudalclan_male_lombard_independent = {
+	type = character
+	gender = male
+	special = holder
+	tier = duchy
+	priority = 29
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+duke_feudalclan_female_lombard_independent = {
+	type = character
+	gender = female
+	special = holder
+	tier = duchy
+	priority = 29
+	cultures = { lombard } 
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+duchy_feudalclan_lombard_independent = {
+	type = title
+	priority = 29
+	tier = duchy
+	cultures = { lombard }
+	governments = { feudal_government clan_government }
+	top_liege = no
+	only_independent = yes
+}
+title_prince_male_lombard = {
+	type = character
+	gender = male
+	special = ruler_child
+	tier = { kingdom empire }
+	priority = 43
+	cultures = { lombard } 
+	governments = { feudal_government clan_government tribal_government }
+	top_liege = no
+}
+title_prince_female_lombard = {
+	type = character
+	gender = female
+	special = ruler_child
+	tier = { kingdom empire }
```

Comment:
Does this really work? I'd expect that since tier = kingdom does not have curly brackets, you can't set a set here.

Reply:
Oh, @Gr770 asked the same thing, funny)

Reply:
Doesn't work. Changed it.